docs/_conversion/steps.md

```diff
@@ -0,0 +1,88 @@
+
+When you’re done configuring your Zap, click “Turn on Zap” to test it.
+
+![](https://cdn.zappy.app/711c9ea75998ffabadc130127eeaa10d.png)
+
+Next, force your trigger to execute. In the Bitbucket example, I created a new issue.
+
+You should see your action run. To check the details in your task history, click the clock icon in the right sidebar menu to verify that your trigger/action/search is returning the correct data.
+
+![](https://cdn.zappy.app/f6b9865ed7c6fd95d67b36f986ed9d8c.png)
+
+![](https://cdn.zappy.app/22fca8a22adbe6727154374c51c7b586.png)
+
+### If your integration has hidden triggers or actions
+
+It’s important that you don’t make edits to the new copy of your project during the conversion process. To test hidden triggers and actions in new Zaps, you’ll need to change the visibility of the trigger/action/search. To do this, create a copy of your integration and make the visibility changes to that version only to access those triggers/actions/searches to test them in the Zap editor. When your test is complete, you can delete this copy of your integration.  
+
+To create a temporary test copy of your integration:
+
+In the left sidebar menu, click “Versions.” Next to your new integration, click the gear icon and select “Clone.” In the dialog box that appears, click the “Clone to version” dropdown menu and select “patch.” It doesn’t matter which option you select here, since this version will be deleted after your testing is complete. Then click “Clone.” Click “Close” to exit the dialog box.
+
+![](https://cdn.zappy.app/ef97ecfc1a3f9a9642c6baa4084a3780.png)
+
+Change the visibility of of a trigger/action/search so you can make a new test Zap:
+
+To select the test version of your integration, click the “Version” dropdown menu in the upper left, or from the versions section. Then select the new version you just created for this test.
+
+![](https://cdn.zappy.app/9757a3a04626669fe2836229dd7e6b86.png)
+
+In the left sidebar menu, select the hidden trigger or action in your integration. At the bottom of the Settings page click the “Visibility Options” dropdown menu and select “Important (Displayed above the fold in the Zap Editor).” Then click “Save.”
+
+![](https://cdn.zappy.app/11786cd7941cf9eab71dc81d73f244db.png)
+
+### When you’re done testing
+
+If everything works as expected, you’re ready to check this trigger/action/search off the list and repeat this process for the next one. If you see any unexpected results, contact us at partners@zapier.com. 
+
+`<INSERT SCREENSHOT OF SIDEBAR CHECKLIST>`
+
+After you’ve tested all your triggers and actions, you’re ready to update your users’ Zaps with the new integration.
+
+## STEP 3: Update your Users’ Zaps
+
+After you’ve completed testing, update your users Zaps to use the integration you’ve created on the new platform. We’ll provide the status of the operation, letting you know when it’s complete.  If you have a lot of users this may take several hours to complete.  
```

Comment:
```suggestion
After you’ve completed testing, update your users' Zaps to use the integration you’ve created on the new platform. We’ll provide the status of the operation, letting you know when it’s complete.  If you have a lot of users this may take several hours to complete.  
```